src/test/resources/testdata/AppAutomationData.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "AppAutomationData",
```

Comment:
Unnecessary complexity for sample tests

Reply:
Removed  AppAutomationData.json and now passing the data in this format

    @Test(description = "Perform addition of two no. on calculator", groups = {"visual"})
    public void additionTest() {
        appSteps.performOperation("2", "plus", "5");
        appSteps.verifyResult("7");
    }